content/en/serverless/guide/datadog_forwarder_node.md

```diff
@@ -84,17 +84,20 @@ Transform:
       env: "<ENV>" # Optional
 ```
 
-Replace `<SERVICE>` and `<ENV>` with appropriate values, `<LAYER_VERSION>` with the desired version of Datadog Lambda layer (see the [latest releases][4]), and `<FORWARDER_ARN>` with Forwarder ARN (see the [Forwarder documentation][2]).
+To fill in the placeholders:
 
-If your Lambda function is configured to use code signing, you must add Datadog's Signing Profile ARN (`arn:aws:signer:us-east-1:464622532012:/signing-profiles/DatadogLambdaSigningProfile/9vMI9ZAGLc`) to your function's [Code Signing Configuration][5] before you can use the macro.
+- Replace `<LAYER_VERSION>` with the desired version of the Datadog Lambda layer. The latest version is `{{< latest-lambda-layer-version layer="node" >}}`.
```

Comment:
does using the version variable work in the docs?

Reply:
It totally does! I didn't realize we had implemented this as a partial. So cool.

Reply:
oooh you implement it in this PR. If I scrolled before commenting, I would be dangerous ;)